tools/vmstart/vmstart.go

```diff
@@ -30,6 +37,8 @@ var (
 	remoteInstanceName = flag.String("remote_instance_name", "", "The remote_instance_name for caching snapshots")
```

Comment:
nit: update the docs to mention that it's for caching snapshots and also for fetching the action, if applicable

Reply:
Done.